### DISCUSSION TITLE
Implementation of Windows audio capture

### DISCUSSION THREAD

**Discussion by lostmsu** [LizardByte/community#37] [Sunshine] [answered]
Can anyone point me into the direction of the subj?

Specifically, the part of the code that takes the output device to capture game sound from, and the part of the code that mixes in the mic.

I am working on a different project, just want to get a general idea of how this is done on Windows.

**Comment by ReenigneArcher** [comment, marked as answer]
https://github.com/SunshineStream/Sunshine/blob/master/sunshine/audio.cpp

https://github.com/SunshineStream/Sunshine/blob/master/sunshine/platform/windows/audio.cpp
